src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.js

```diff
@@ -491,6 +491,22 @@ export default function ApplicationForm({
     return re.test(postalCode.toUpperCase());
   };
 
+  const scrollToRef = ref => {
+    if (ref !== null && ref.current !== null) {
```

Comment:
should just be able to do
`if (ref && ref.current)`

Reply:
Is there a benefit to using `if (rev && ref.current)` versus `if(ref !== null && ref.current !== null)`?

Reply:
yes, in JS values can be falsey or truthy
so the usual way to check for falsey values such as `null` is:

```
if (!a) {
    // `a` is falsey, which includes `undefined` and `null`
    // (and `""`, and `0`, and `NaN`, and [of course] `false`)
}
```

not only is it simpler, but this way you reject all falsey values

Reply:
Good to know, I'll change it.